src/content/resources/faq.md

```diff
@@ -521,12 +528,19 @@ between physical pixels and logical CSS pixels.
 
 ### What kind of app performance can I expect?
 
-You can expect excellent performance. Flutter is
+In general, you can expect excellent performance. Flutter is
 designed to help developers easily achieve a constant 60fps.
 Flutter apps run using natively compiled code&mdash;no
 interpreters are involved.
 This means that Flutter apps start quickly.
 
+Flutter's performance when using native code depends on your app's
+architecture. For optimal performance, familiarize yourself with Flutter's
+[platform channels][]. These channels provide an asynchronous message-passing
+system for communicating with native code.
+
+[platform channels]: /platform-integration/platform-channels
```

Comment:
Should we also add a link to "how to architect a Flutter app", Eric's new section? I defer to you.